src/e2e/features/1-authentication.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Login
 
   Scenario: Bob wants to login to ZAC
-    Given "Bob" navigates to "zac" with path "/"
+    Given "Bob" navigates to "zac" with path "/" but other destination is expected
```

Comment:
What does this mean: "other destination is expected"? If Bob expects another destination then why does he navigate to "/"? :-) I don't quite understand this I guess.

Reply:
I have removed the step, because it wa snot solving anything